ietf/static/ietf/js/edit-meeting-schedule.js

```diff
@@ -164,6 +192,55 @@ jQuery(document).ready(function () {
         }
     }
 
+    // uses current time if now is not passed in
+    function isFutureTimeslot(timeslot, now) {
+        // resist the temptation to use native JS Date parsing, it is hopelessly broken
```

Comment:
I'm curious to read a description of how it's broken.  Do you have a handy reference?

Reply:
It's mentioned here https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/parse 

> It is not recommended to use Date.parse as until ES5, parsing of strings was entirely implementation dependent. There are still many differences in how different hosts parse date strings, therefore date strings should be manually parsed (a library can help if many different formats are to be accommodated).

Basically, handling of time zones, DST, etc, can vary.